Testscripts/Linux/DM_HotAdd_Verify_udev.sh

```diff
@@ -34,18 +26,50 @@ if [ -e summary.log ]; then
     rm -rf summary.log
 fi
 
+config_path=$(get_bootconfig_path)
+
+# /sys/devices/system/memory/auto_online_blocks = online => when the memory is hot-added, it will be online automatically
+if grep CONFIG_MEMORY_HOTPLUG_DEFAULT_ONLINE=y $config_path; then
+    LogMsg "CONFIG_MEMORY_HOTPLUG_DEFAULT_ONLINE=y is set in $config_path, then no need to check udev rules on $DISTRO"
+    value_of_aob=$(cat /sys/devices/system/memory/auto_online_blocks)
+    if [ "$value_of_aob" == "online" ]; then
+        LogMsg "Value of /sys/devices/system/memory/auto_online_blocks is expected - online"
+        SetTestStateCompleted
+        exit 0
+    else
+        LogErr "Value of /sys/devices/system/memory/auto_online_blocks is unexpected - $value_of_aob"
+        SetTestStateFailed
+        exit 0
+    fi
+fi
+
 # Search in /etc/udev and /lib/udev folders
 for udevfile in $(find /etc/udev/ /lib/udev/ -name "*.rules*"); do # search for all the .rules files
     match_count=0
-    for i in "${items[@]}"
+    for i in "${items2[@]}"
+    do
+        grep "$i" "$udevfile" > /dev/null # grep for the udev rule
```

Comment:
Can we use the same pattern of condition check (if cond) as done in line no 32 (```grep CONFIG_MEMORY_HOTPLUG_DEFAULT_ONLINE=y```)